full-stack-tests/core/src/backend/RpcImpl.ts

```diff
@@ -23,11 +23,25 @@ async function initializeContainer(containerId: string) {
   await ViewStore.CloudAccess.initializeDb({ baseUri: AzuriteTest.baseUri, storageType, containerId, accessToken });
 }
 
-export class TestRpcImpl extends RpcInterface implements TestRpcInterface { // eslint-disable-line deprecation/deprecation
+export class TestRpcImpl extends RpcInterface implements TestRpcInterface {
   public static register() {
     RpcManager.registerImpl(TestRpcInterface, TestRpcImpl);
   }
 
+  public async openSnapshot(filePath: string): Promise<IModelConnectionProps> {
```

Comment:
This function appears to be a copy of `IpcHost.openSnapshot`, but without the optional open options. Since it doesn't appear that this function or `IpcHost.openSnapshot` makes use of their `this` pointers, it seems like having a public static function in `IpcHost` that both non-static openSnapshot functions call would be better, to avoid code copying. (Maybe I'm misreading these two functions.)

Reply:
I would like to avoid increasing `IpcHost` API size just to avoid some code duplication in tests.

Additionally, `IModelHost.snapshotFileNameResolver` may be deprecated soon. If it is deprecated, this function would be the first to be refactored while `IpcHost.openSnapshot` would need to keep supporting it until deprecated property is removed.